src/eastwood/lint.clj

```diff
@@ -839,7 +840,7 @@ Eastwood has other forms of effective, safe parallelism now. Falling back to seq
       (reporting/note reporter "== No namespaces were linted. This might indicate a misconfiguration."))
     {:some-warnings (or (> warning-count 0)
                         has-errors?
-                        nothing-was-linted?)
+                        (and (zero? modified-since) nothing-was-linted?))
```

Comment:
I'd suggest to extract the `:modified-since 0` to `:modified-since default-modified-since` and use `(= modified-since default-modified-since)` here

Reply:
It's funny you should mentioned Polylith, because we're using it. A long linting run in our case is around 3mins for 200 files. We do run Eastwood in parallel with our github matrix projects tests, exactly as you describe it, by determining which projects need building using Polylith, and only building those. And, you're right, Eastwood is not usually the slowest cog either. We was just optimizing our overall build process, the caching is something we thought worth adopting, and was just surprised when the build step failed due to the exit code. Thanks for looking through the PR, and I shall make the change you suggested. Cheers!

Reply:
3 mins sounds pretty sweet!

The worst times I've witnessed over the years were around the 7-minute mark, maybe 9. Think truly enterprise-sized behemoths.

But if you can optimize things beyond 3m, all the power to you 👍